frontend/src/PipelineValidation.js

```diff
@@ -293,16 +294,68 @@ class UnionRule {
     }
 }
 
+// Character-class flags
+const ValueRuleFlags = {
```

Comment:
What is this for?

Reply:
There is nothing using these validations in this PR, so remove it please. We can add something like this once it actually becomes needed, although I would prefer a more composable solution.